chapter_linear-networks/linear-regression-gluon_vn.md

```diff
@@ -325,7 +325,10 @@ To access each parameter's values as an `ndarray`, we invoke its `data` method.
 As in our from-scratch implementation, note that our estimated parameters are close to their ground truth counterparts.
 -->
 
-*dịch đoạn phía trên*
+Dưới đây, chúng ta so sánh các tham số của mô hình đã học thông qua việc huấn luyện trên tập dữ liệu hữu hạn và các tham số thực được sinh ra từ tập dữ liệu của chúng ta. 
+Để truy xuất các những tham số có Gluon, trước hết chúng ta đọc từ lớp 'net', rồi truy cập đến trọng số của lớp đó ('weight') và phân cực ('bias').
+Để truy xuất giá trị mỗi tham số dưới dạng một mảng 'ndarray', ta sử dụng phương thức 'data'.
+Để ý như trong phần hiện thực khái lược này của chúng ta, các tham số được ước lượng gần với các nhãn gốc tương ứng của chúng.
```

Comment:
```suggestion
Theo sự khởi tạo ban đầu thì ta cần chú ý đến các tham số được ước lượng gần với các tham số tương ứng của chúng.
```

Reply:
Câu này mình bảo lưu theo ý của tác giả, câu văn bạn dịch nó không mang ý tác giả muốn truyền tải.  Vì là các tham số ta khởi tạo thì không cách nào ta biết nó gần với các tham số tương ứng của nó cả. Câu văn này vô nghĩa.

Reply:
Tác giả dùng từ from-scratch có ý nói là cái code này chỉ để mô tả ý tưởng nó chưa được tỉa tót tối ưu chứ không có nghĩa là ban đầu hay bắt đầu gì ở đây.

Reply:
@LongNguyen1984 hi bạn, `implementation from scratch` có ý là code từ đầu nhé, tức mình tự cài đặt thuật toán chỉ dùng các `ndarray` thôi. Còn khi dùng gluon thì các phép toán đã được abstract rồi.

Bạn thử tham khảo lại suggestion của mình nhé, nếu bạn thấy có gì không đúng thì mình có thể thảo luận
```suggestion
Để ý rằng các tham số được ước lượng khá gần với các tham số thực sự, giống với lúc xây dựng mô hình từ đầu.
```

Reply:
@LongNguyen1984 `from-scratch` có nghĩa là `làm từ đầu`. Làm từ đầu nhưng vẫn hoàn chỉnh, chạy được chứ không phải chỉ đơn giản là mô tả.
```suggestion
Giống như cách xây dựng từ đầu, các tham số ước lượng tìm được gần với giá trị chính xác của chúng.
```
`As` cũng có nghĩa là `giống với`.